tests/e2e/system_test/test_cr_resources_validation.py

```diff
@@ -14,55 +18,101 @@
 ERRMSG = "Error in command"
 
 
-@system_test
+@tier3
 @skipif_ocp_version("<4.13")
 @skipif_ocs_version("<4.13")
 class TestCRRsourcesValidation(E2ETest):
     """
     Test that check that csi addons resources are not editable after creation
     """
 
-    def test_network_fence_not_editable(self):
-        """
-        Test case to check that network fence object is not editable once created
-        """
+    def setup(self):
```

Comment:
Please implement a teardown method that will bring everything back to how it was prior to the test, in case of a premature test failure